### ISSUE TITLE
Consolidate customization docs

### ISSUE THREAD

**Issue by plata** [ublue-os/website#628]
Currently, there are several pages around the topics "customization"/"building your own":
- https://universal-blue.org/guide/fork-your-own/
- https://universal-blue.org/CONTRIBUTING/#make-your-changes
- https://universal-blue.org/tinker/modification/

It would be good to streamline this such that there's a consolidated way to build custom uBlue variants (why isn't e.g. Bazzite based on startingpoint?).

My suggestion would be:
- remove "Fork Your Own" from "User's Guide" (use "Tinkerer's Guide" only)
- move common documentation (e.g. how to build+test images locally) to a "Development" section

Maybe it would even make sense to completely split user and developer (i.e. contributor/tinkerer) documentation completely.

**Comment by essn** [comment]
I would create a new issue, but I'll hop into this thread because it relates to consolidation and documentation.

I recently ran into trouble going between `latest` and `gts`, not realizing that `gts` meant:

> After some hard work by our team members we were able to generate F39 builds of our images. Of course they are not stable yet, but if you feel the need you can already try them out. While working on this, we also introduced the gts tag which will enable you to cruise along on a steady stable pace.

Searching on https://universal-blue.org/ led me to one blog post - https://universal-blue.org/blog/posts/ublue-f39-builds/?h=gts. I discovered that then dug around the builds and figured it out.

It's relatively easy; the resources are provided, but I first looked at https://universal-blue.org/images/#full-image-list. 

Any thoughts on the list of suffixes being `gts`, `main`, `nvidia`? The image list is the first place I would (did) look.
